build/yamls/windows/containerd-with-ovs/conf/Install-OVSDriver-Containerd.ps1

```diff
@@ -0,0 +1,18 @@
+$ErrorActionPreference = "Stop"
+mkdir -force c:/openvswitch/driver
+$mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+$mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+$OVSDriverDir = "$mountPath\openvswitch\driver"
+
+# Check if OVSExt driver is already installed
+$driverStatus = netcfg -q ovsext
+if ($driverStatus -like '*not installed*') {
+  # Install OVS Driver
+  $result = netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
+  if ($result -like '*failed*') {
```

Comment:
($result -like '*failed*') is checking whether the variable $result contains the word "failed",  it will enter here only when error message contains failed, we might want to add another else to capture any unknown errors
 else {
        # Handle unexpected output
        Write-Host "Unexpected output from netcfg: $result"
        exit 1
    }
and can prefer to move Write-Host "OVSExt driver has been installed" at the end